gcp-pubsub/build.gradle

```diff
@@ -3,15 +3,18 @@ plugins {
 }
 
 dependencies {
+    annotationProcessor(mnSerde.micronaut.serde.processor)
+    implementation(mnSerde.micronaut.serde.support)
```

Comment:
is this necessary?

Reply:
I have removed those https://github.com/micronaut-projects/micronaut-gcp/pull/848/commits/f4fbc94a86a8b93be0ad66f6e70459cc3abb1dc3